build.gradle

```diff
@@ -442,6 +443,31 @@ project(':iceberg-spark-runtime') {
   }
 }
 
+project(':iceberg-flink') {
+  apply plugin: 'com.commercehub.gradle.plugin.avro'
+
+  dependencies {
+    compile project(':iceberg-api')
+    compile project(':iceberg-core')
+    compile project(':iceberg-data')
+    compile project(path: ':iceberg-data', configuration: 'testArtifacts')  // TODO: to remove, for RandomGenericData
+    compile project(':iceberg-hive')
+    compile project(':iceberg-parquet')
+
+    compileOnly("org.apache.flink:flink-streaming-java_2.11")
```

Comment:
maybe make the Scala version a gradle property. unfortunately, Flink doesn't have a clean Java API. even `flink-streaming-java` has a Scala version suffix in the jar name.

@rdblue what is your take on Scala cross compile? I saw the `iceberg-spark`  module also has hard-coded `2.11` Scala suffix in the spark deps